src/main/scala/io/qbeast/spark/table/IndexedTable.scala

```diff
@@ -257,25 +262,18 @@ private[table] class IndexedTableImpl(
       while (tries > 0) {
         val announcedSet = write.announcedCubes.map(indexStatus.revision.createCubeId)
         val updatedStatus = indexStatus.addAnnouncements(announcedSet)
-        val replicatedSet = updatedStatus.replicatedSet
-        val revisionID = updatedStatus.revision.revisionID
         try {
```

Comment:
Then is it necessary to keep the try, catch?

Reply:
I can imagine a conflict if two concurrent transactions try to create a new revision while writing the data.